rubocop.yml

```diff
@@ -326,6 +326,9 @@ Metrics/BlockLength:
 Metrics/ClassLength:
   Enabled: false
 
+Metrics/CollectionLiteralLength:
+  Enabled: true
```

Comment:
https://github.com/rubocop/rubocop/pull/11584#issuecomment-1434147644 summarizes why I created this cop and why I'd like to enable it. TL;DR: every time I've seen huge literals, there was a better approach.